test/e2e/logs_test.go

```diff
@@ -65,7 +65,11 @@ var _ = Describe("Podman logs", func() {
 			Expect(logc).To(Exit(0))
 			cid := logc.OutputToString()
 
-			results := podmanTest.Podman([]string{"logs", cid})
+			results := podmanTest.Podman([]string{"wait", cid})
```

Comment:
What is special about this test? The other test below use the same logic without wait as well.

You could also just remove the `-d` from podman run instead of the extra wait call.

Reply:
What's special is that this is another [podman logs flake](https://api.cirrus-ci.com/v1/artifact/task/5355295198150656/html/int-podman-fedora-37-root-container.log.html#t--all-lines--json-file--1). I haven't seen flakes in those other tests, so I didn't look at them.

Removing `-d` was my first choice, but I realized I don't know enough about logging to know if it behaves differently under `-d`. I chose to assume the test author used it that way for a reason.

Reply:
Logging doesn't behave differently.
But if you use `-d` we will detach directly and not wait for exit, without `-d` we will wait for exit, thus `podman wait` would not be needed in that case.